inlong-sort/sort-formats/format-base/src/main/java/org/apache/inlong/sort/formats/base/DefaultTableFormatSerializer.java

```
@@ -67,15 +52,14 @@ public void flatMap(
         try {
             bytes = serializationSchema.serialize(row);
         } catch (Exception e) {
-            if (ignoreErrors) {
```

sorry why remove this ignore errors?

---

done